examples/hot-reload/README.md

```diff
@@ -7,9 +7,12 @@ Application demonstrating the file synchronization mode with both NodeJS and Pyt
 #### Init
 
 ```bash
-skaffold dev
+skaffold dev [--default-repo docker.io/my-repo]
 ```
 
+If you're not running against a local cluster like `minikube` or `kind`, then you'll need to specify the `--default-repo` flag to push to an image repository of your own.
+The default value for `--default-repo` is `docker.io/library/`.
```

Comment:
This isn't correct. The behaviour you're seeing is a result of Docker's naming conventions.
A simple image name like `openjdk` or `golang` (doesn't have a registry or repository) is implicitly considered to be a [Docker Official Image](https://docs.docker.com/docker-hub/official_images/), which are located at `docker.io/library/<image>`.

We could have an explanation here, but it would need to be copied across the examples.  It's better to put the description in `examples/README.md` instead.

Reply:
ah i see it's already in `examples/README.md`. thanks, I didn't see that.